### ISSUE TITLE
Xlights crashes when resizing/moving windows in sequence tab

### ISSUE THREAD

**Issue by bluecreeklightguy** [smeighan/xLights#2238]
**Describe the bug**
When I resize/move windows in the Sequencer tab, Xlights crashes.

**To Reproduce**
1. Go to sequence tab.
2. Choose a Perspective.  In my video, I choose a single screen version (specifically the one called "No 2nd Monitor"), but I've reproduced it with two screens as well.
3. Select View -> Windows -> Reset to Defaults.  Note that I needed to do this in my video in order to get my second (called Neighborhood) preview on screen.
4. Attempt to move or resize the preview window.  Note that I was also able to generate a crash by resizing other windows, such as the Model Preview window.
5. After a few attempts (this is a bit random, but never takes me more than about 5-10 seconds of resize/move), Xlights will crash.

I have attached the .zip file, and a video that shows how I create the problem.  It is not necessary for a sequence to be open to demonstrate the issue.  The crash happens whether there is a sequence open or not.

**Expected behavior**
Expected behavior would allow me to resize and reposition all sequencer windows on my screen(s) without a crash.

**Screenshots**

**Versions (please complete the following information):**
 - OS: Windows 10
 - xLights version:  2020.39 (although I also reproduced this problem in 2019.64 and 2019.74)

**Attachments**
- zip file containing .xml files and the crash log
[xLights_dbgrpt-13540-20200923T004827.zip](https://github.com/smeighan/xLights/files/5265962/xLights_dbgrpt-13540-20200923T004827.zip)

- link to video file showing how I am able to produce the crash.  The demonstration of the crash begins around the 1:12 mark.
https://youtu.be/lkePlbJ7uc8

**Comment by keithsw1111** [comment]
This is 90% likely a video driver issue. Update your drivers to the latest available from your card supplier (not Microsoft).

**Comment by bluecreeklightguy** [comment]
Hi, Keith.  Thanks for the suggestion, as I'd used the Windows drivers for my prior updates.  I updated the drivers tonight from NVIDIA and Intel (one for external display; one for laptop display screen).  Unfortunately, xlights still crashes.  

I didn't include it in my original writeup, but thought I'd mention another version of the crash that I experience as well...  If I'm opening the Sequencer tab and the prior Perspective was a two-screen perspective and I have my Neighborhood preview up for display in that perspective, the crash occurs immediately (doesn't complete the drawing of all of the contents of the windows on screen) and doesn't generate a crash report.  Maybe also of note is that these crashes didn't occur until I added this year's pixel additions and don't occur if I go back to a prior year's layout with a lower channel count.

Any items I might be able to try to help narrow down the issue?

**Comment by bluecreeklightguy** [comment]
Just did a little more troubleshooting.  I remembered that Windows had recently performed an automatic update.  I rolled back to the restore point that occurred before the update.  I shouldn't be surprised, but things are working again.  I have no idea what the hell Windows changed that caused problems, as even with the most up-to-date drivers, I was experiencing the crash.

Sorry if I sent you down a rabbit hole.  I didn't think to double-check the info. on the specific Windows update so I could provide that to you, but I'll watch for it next time (if there is one).  Let me know if there's any other information you'd like to have for potential compatibility/troubleshooting.

Thanks for all your time and effort on the software.  I really appreciate it.
